gce/Makefile

```diff
@@ -173,8 +213,12 @@ gce-create: kubectl calicoctl $(TEMPLATED_OUTPUT)
 	  --metadata-from-file user-data=build/node-config-template.yaml \
 	  --no-address \
 	  --tags no-ip
+		echo "Waiting for creation of worker nodes to finish..."; \
```

Comment:
You've got multiple tabs here.  I think you should use one tab then spaces, to match the rest of the file.